test/sagemaker_tests/pytorch/training/resources/mnist/smmodelparallel_pt_mnist.py

```diff
@@ -217,14 +220,12 @@ def get_parser():
         default=False,
         help="For Saving the current Model",
     )
-    parser.add_argument("--num-microbatches", type=int, default=4)
     parser.add_argument("--num-batches", type=int, default=0)
-    parser.add_argument("--num-partitions", type=int, default=2)
-    parser.add_argument("--horovod", type=int, default=0)
-    parser.add_argument("--ddp", type=int, default=0)
     parser.add_argument("--amp", type=int, default=0)
-    parser.add_argument("--pipeline", type=str, default="interleaved")
     parser.add_argument("--assert-losses", type=int, default=0)
+    parser.add_argument("--data-dir", type=str, default=None)
+    parser.add_argument("--ddp", type=int, default=0)
+    parser.add_argument('--mp_parameters', type=str, default='')
```

Comment:
Will this change be backward compatible with sm-mp tests on PT 1.7 and PT 1.6, which use smp-1.2.0?

Reply:
Yeah, all changes here are backwards compatible. I just updated the script to the latest in our repo. We run 1.6 and 1.7 tests in our internal pipelines.